.github/workflows/unit-tests.yml

```diff
@@ -7,15 +7,23 @@ env:
 jobs:
   test:
     name: "Tests"
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        config:
+          # Linux
+          - { os: ubuntu-latest, toolchain: stable, target: x86_64-unknown-linux-gnu, cross: false }
+          # Macos
+          - { os: macos-latest, toolchain: stable, target: x86_64-apple-darwin, cross: false }
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
 
       - name: Rust setup
         run: |
           bash ./scripts/setup/dev_setup.sh
-
+          bash ./scripts/setup/dev_setup.sh -o
```

Comment:
Why do we need to install twice with and without `-o`?

Reply:
https://github.com/datafuselabs/datafuse/blob/master/scripts/setup/dev_setup.sh#L123, github macOS runner did not install mysql client by default(https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md), thus we need to install mysql client manually

Reply:
Just a single `bash ./scripts/setup/dev_setup.sh -o` is not enough? I am supposing the `dev_setup.sh -o` won't turn off anything that are installed by `dev_setup.sh`. 🤔

Reply:
first setup would execute the following flow:
https://github.com/datafuselabs/datafuse/blob/master/scripts/setup/dev_setup.sh#L222-L228
enable operation install means that it would only install operator dependencies such as python, mysql client etc.
a better approach may be the following command
```bash
bash ./scripts/setup/dev_setup.sh -o -t
```

Reply:
Hm... Thanks for your explanation! The control-flow in this script is not that intuitive right?

Reply:
help messages already shown in script usage pages, maybe we could add more documentation on setup part(an issue?)

Reply:
🤔 I've read this script several times before and I did notice there is a `usage` section. :(((

The help message may need to explain what it does when no options supplied?

Reply:
agreed, help message could provide some example use cases for the setup script